### PR TITLE
adding enumerations support.

### PR DESCRIPTION
**Pull request by MohammadAlhallaq**
hello.
I've just added support to Enumerations classes.

### CODE REVIEW

**Review by MohammadAlhallaq — COMMENTED** [review]
changed the variable name to match the returned value.

File: src/Commands/MakeEnumCommand.php [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+namespace JustinByrne\ExtraArtisanCommands\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Filesystem\Filesystem;
+
+class MakeEnumCommand extends Command
+{
+    public $signature = "make:enum {name} {type=string}";
+
+    public $description = "Create a new enum class";
```
Comment (JustinByrne) [inline src/Commands/MakeEnumCommand.php:12]: Maybe change this from `Create a new enum class` to `Create a new enum`

**Comment by JustinByrne** [comment]
Just need to add the command to the README file.

**Review by JustinByrne — APPROVED** [review]
Brilliant, thank you.

**Pull request merged**